haiku-misc/gadgeteer/gadgeteer-0.8.1.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Command line for creating GUI forms"
+DESCRIPTION="Gadgeteer is a command line app for creating small GUI forms \
+that users can fill out or play with. Once the users press the \"DONE\" \
+button at the bottom of the screen, the command returns the user's input \
+values so they could be used for different purposes."
+HOMEPAGE="http://www.goodeveca.net/beos/"
```

Comment:
Oops, you forgot to switch to https. (https://github.com/haikuports/haikuports/pull/3399#discussion_r235504991)
(That's not strictly required, but since the site is also available through https, using that protocol is preferred.)

Reply:
Oops. Will change this right away :)